src/test/java/org/openmrs/contrib/qaframework/automation/EditPatientRelationshipSteps.java

```diff
@@ -28,26 +37,31 @@ public void destroy() {
 
 	@Given("System loads clinicianFacingPatientDashboardPage")
 	public void loadClinicianFacingPatientDashboardPage() throws Exception {
+		matchPatientIds(patientDashboardId);
 
- 
 	}
+
 	@When("User clicks on editRegistrationInformation")
 	public void clickEditRegistrationInformation() {
-
+		dashboardPage.goToRegistrationSummary().waitForPage();
 	}
-	@And("User clicks on EditRelationships")
-	public void clickEditRelationships() {
+
+	@And("User clicks on RegistrationSummary")
+	public void clickOnRegistrationSummary() {
+		RegistrationSummaryPage summary = new RegistrationSummaryPage(page);
+		summary.goToEditPatientRelationship();
```

Comment:
which page is this parenting RegistrationSummaryPage, it needs to be properly named

Reply:
RegistrationSummaryPage is being used with in distro , so am importing it from distro package